### ISSUE TITLE
Crash on Rename or Teleport dialogue open

### ISSUE THREAD

**Issue by thetechnodragon** [DArkHekRoMaNT/TeleportationNetwork#1]
Vintage story 1.14.5
TPNetwork 1.0.2
multiplayer server continues running
client crashes

```
-------------------------------
Version: v1.14.5 (Stable)1/10/2021 4:32:20 PM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object
  at TeleportationNetwork.GuiDialogTeleport.OnMouseMove (Vintagestory.API.Client.MouseEvent args) [0x00065] in <07765ef517dc4ce0ab463dcc41a3a360>:0 
  at Vintagestory.Client.NoObf.GuiManager.OnMouseMove (Vintagestory.API.Client.MouseEvent args) [0x0002f] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientMain.OnMouseMove (Vintagestory.API.Client.MouseEvent args) [0x000d2] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at _Vq7Xo4xIdbZDwh8iDcGnovGKH7D._IM41PBlAoi8uHXSL5o1MkAYJlhV (Vintagestory.API.Client.MouseEvent ) [0x00013] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at _7KGTxxuvfNubJZCLAMEMJZBkPDeA._IM41PBlAoi8uHXSL5o1MkAYJlhV (Vintagestory.API.Client.MouseEvent ) [0x00018] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientPlatformWindows.UpdateMousePosition () [0x00087] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame (System.Object sender, OpenTK.FrameEventArgs e) [0x000c3] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[OpenTK.FrameEventArgs].invoke_void_object_TEventArgs(object,OpenTK.FrameEventArgs)
  at OpenTK.GameWindow.OnRenderFrame (OpenTK.FrameEventArgs e) [0x00000] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.OnRenderFrameInternal (OpenTK.FrameEventArgs e) [0x00010] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.RaiseRenderFrame (System.Double elapsed, System.Double& timestamp) [0x0000c] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.DispatchRenderFrame () [0x00038] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.Run (System.Double updates_per_second, System.Double frames_per_second) [0x00104] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.Run () [0x00000] in <7ef43334b29e406ba375449554b0451a>:0 
  at _4eRDDOqtbaPbgjN2dJ1zzD1FCRy._ExJp6VhLWG4xQoHsVn72tgyNbeG (_bWl72q5sGuTQypu8W8r7cRXUcRF , System.String[] ) [0x003b7] in <b89e038b0c1344a5adf1089e1af77db4>:0 
  at _4eRDDOqtbaPbgjN2dJ1zzD1FCRy+_BoG0j42BkitB8Sh3DeEI5jZLAl9._yd95GhMXBTaLXtVhTFUFRXqwZfA () [0x00000] in <b89e038b0c1344a5adf1089e1af77db4>:0 
  at _XKAWv6MyciUiZmECUonQCwPaCwE._ExJp6VhLWG4xQoHsVn72tgyNbeG (System.Threading.ThreadStart ) [0x00007] in <5beff6cd6c604ff79cc409db0e91b473>:0 
-------------------------------
Version: v1.14.5 (Stable)1/10/2021 4:36:30 PM: Critical error occurred
System.Collections.Generic.KeyNotFoundException: The given key '501069, 111, 508195' was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <671ef0784b04474c83bb500a849492f2>:0 
  at Vintagestory.GameContent.GuiDialogRenameTeleport.get_Data () [0x00011] in <07765ef517dc4ce0ab463dcc41a3a360>:0 
  at Vintagestory.GameContent.GuiDialogRenameTeleport..ctor (System.String DialogTitle, Vintagestory.API.MathTools.BlockPos blockEntityPos, Vintagestory.API.Client.ICoreClientAPI capi, Vintagestory.API.Client.CairoFont font) [0x00237] in <07765ef517dc4ce0ab463dcc41a3a360>:0 
  at TeleportationNetwork.BlockEntityTeleport.OnShiftRightClick () [0x0003e] in <07765ef517dc4ce0ab463dcc41a3a360>:0 
  at TeleportationNetwork.BlockTeleport.OnBlockInteractStart (Vintagestory.API.Common.IWorldAccessor world, Vintagestory.API.Common.IPlayer byPlayer, Vintagestory.API.Common.BlockSelection blockSel) [0x00053] in <07765ef517dc4ce0ab463dcc41a3a360>:0 
  at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseBlock (Vintagestory.API.Common.Block selectedBlock, Vintagestory.API.Common.BlockSelection blockSelection) [0x00000] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected (System.Single dt) [0x00371] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.UpdatePicking (System.Single dt) [0x0009e] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame (System.Single dt) [0x0004a] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.API.Client.DummyRenderer.OnRenderFrame (System.Single deltaTime, Vintagestory.API.Client.EnumRenderStage stage) [0x00000] in <a9cc842435f44963aecddaa846ea4efb>:0 
  at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage (Vintagestory.API.Client.EnumRenderStage stage, System.Single dt) [0x00072] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage (Vintagestory.API.Client.EnumRenderStage stage, System.Single dt) [0x00028] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer (System.Single dt) [0x00092] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at _Vq7Xo4xIdbZDwh8iDcGnovGKH7D._df3GxtG5OOiD7YZvbaraCiLbckx (System.Single ) [0x0003e] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at _7KGTxxuvfNubJZCLAMEMJZBkPDeA._07k4LSTgxALpYbS1S6K07SItFcH (System.Single ) [0x001b4] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at _7KGTxxuvfNubJZCLAMEMJZBkPDeA._aysJzMcCvicDqo8MC3TmjcDsfdZ (System.Single ) [0x000d4] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame (System.Object sender, OpenTK.FrameEventArgs e) [0x00147] in <5beff6cd6c604ff79cc409db0e91b473>:0 
  at (wrapper delegate-invoke) System.EventHandler`1[OpenTK.FrameEventArgs].invoke_void_object_TEventArgs(object,OpenTK.FrameEventArgs)
  at OpenTK.GameWindow.OnRenderFrame (OpenTK.FrameEventArgs e) [0x00000] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.OnRenderFrameInternal (OpenTK.FrameEventArgs e) [0x00010] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.RaiseRenderFrame (System.Double elapsed, System.Double& timestamp) [0x0000c] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.DispatchRenderFrame () [0x00038] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.Run (System.Double updates_per_second, System.Double frames_per_second) [0x00104] in <7ef43334b29e406ba375449554b0451a>:0 
  at OpenTK.GameWindow.Run () [0x00000] in <7ef43334b29e406ba375449554b0451a>:0 
  at _4eRDDOqtbaPbgjN2dJ1zzD1FCRy._ExJp6VhLWG4xQoHsVn72tgyNbeG (_bWl72q5sGuTQypu8W8r7cRXUcRF , System.String[] ) [0x003b7] in <b89e038b0c1344a5adf1089e1af77db4>:0 
  at _4eRDDOqtbaPbgjN2dJ1zzD1FCRy+_BoG0j42BkitB8Sh3DeEI5jZLAl9._yd95GhMXBTaLXtVhTFUFRXqwZfA () [0x00000] in <b89e038b0c1344a5adf1089e1af77db4>:0 
  at _XKAWv6MyciUiZmECUonQCwPaCwE._ExJp6VhLWG4xQoHsVn72tgyNbeG (System.Threading.ThreadStart ) [0x00007] in <5beff6cd6c604ff79cc409db0e91b473>:0 
-------------------------------
```

**Comment by DArkHekRoMaNT** [comment]
Fixed in v1.0.3

**Comment by DArkHekRoMaNT** [comment]
Perhaps there is still an error when opening the teleport dialogue? Was this fixed in v1.0.3?

**Comment by DArkHekRoMaNT** [comment]
Fixed in v1.0.4
